### ISSUE TITLE
Document or enforce minimum Docker and docker-compose versions

### ISSUE THREAD

**Issue by jimiolaniyan** [dhis2/cli#80]
docker version: `17.09.0-ce`
d2 version: `1.2.4`

The following command from the README
```
d2 cluster up dev --port 8082 --seed
``` 

fails with the message:
```
Invalid interpolation format for "environment" option in service "core": "-Dcontext.path='${DHIS2_CORE_CONTEXT_PATH:-}'"
 
[DEBUG ERROR] docker-compose exited with non-zero exit code (1). 
[DEBUG] tryCatchAsync(exec(docker-compose)) error: undefined 
[ERROR] Failed to spin up cluster docker-compose cluster 
```

There are no such issues with version 18 of docker.

**Comment by amcgee** [comment]
Thanks for mentioning this @jimiolaniyan - we should document somewhere (or enforce at runtime) that the minimum version of `docker` and `docker-compose` are (specifically) `18.06.0` and `1.22.0`, respectively.

**Comment by amcgee** [comment]
@erikarenhill this got stale and was closed, but is the same as #289
